src/xcode/ENA/ENA/Source/Scenes/QRScanner/QRScannerCoordinator.swift

```diff
@@ -364,10 +364,24 @@ class QRScannerCoordinator {
 		_ initializationData: TicketValidationInitializationData
 	) {
 		showActivityIndicator()
+		
+		#if DEBUG
```

Comment:
Is the following code some logic, which one would expect by only debugging (not ui-testing) the application?
In other words, could this logic be a blocker of having a proper debugging with other test data?

Reply:
Good point, I will only restrict to UI testing.